### DISCUSSION TITLE
How to make icons show up in articles directory page?

### DISCUSSION THREAD

**Discussion by wvictor14** [jpanther/congo#894] [Q&A] [answered]
I would like to add the icon thumbnails shown [here](https://jamespanther.com/projects/) and [here](https://jeremic.ca/projects/)

I thought it would be simply including the `icon:` parameter into an articles YAML frontmatter, but it doesn't work.. here is and example: This is the [source](https://github.com/wvictor14/wvictor14.github.io/blob/d36a9b4906c669665632405a83178ce3f8c65894/content/projects/rbiotechsalary.md?plain=1#L3) for an article where I've chosen the "reddit" icon to display. 
```
---
title: rbiotechsalary
icon: reddit
date: 2024-05-15
weight: 41
summary: >
    A web app for querying biotech/pharma salary data. Consists of end-to-end data ingestion, processing, to web app deployment. Built with R Shiny and Docker. [source](https://github.com/wvictor14/rbiotechsalary)
externalUrl: https://victoryuan.com/projects/rbiotechsalary
---
```

But icons do not show up for this article or any others: https://www.victoryuan.com/projects/

**Comment by stereobooster** [comment]
On the same note. It would be nice to have template / page for projects. Maybe 2 (?) column grid with cards. Card can have name (link), description, icon/image, maybe icons witch counters (github stars, npm downloads, etc). 

Here are some examples:
- https://akashrajpurohit.com/projects/
- https://antfu.me/projects

**Comment by stereobooster** [comment, marked as answer]
To answer your question, @wvictor14. You also need to add this icon to `assets` folder

<img width="299" alt="Screenshot 2024-06-15 at 14 53 32" src="https://github.com/jpanther/congo/assets/179534/3c2771ec-6a13-4270-bfe8-0a653f3056c6">

<img width="266" alt="Screenshot 2024-06-15 at 14 53 52" src="https://github.com/jpanther/congo/assets/179534/c4321d70-cbea-4e1e-850a-687a5bf7d3ac">

**UPD** also you need [`layouts/projects/list.html`](https://github.com/fjeremic/fjeremic.github.io/blob/master/layouts/projects/list.html)

**Reply by wvictor14** [reply]
> **UPD** also you need [`layouts/projects/list.html`](https://github.com/fjeremic/fjeremic.github.io/blob/master/layouts/projects/list.html)

The info I needed, thanks!
